src/standard/index.d.ts

```diff
@@ -37,7 +37,17 @@ export declare interface StripeBackendAPI {
    * @returns a Promise that resolves on success and rejects on failure.
    *     Any error should be reported as a string that can be displayed to the user.
    */
-  completeCharge(sourceID: string, amount: number, shippingMethod?: StripeShippingMethod, shippingAddress?: StripeAddress): Promise<void>;
+  capturePayment(sourceID: string, amount: number, shippingMethod?: StripeShippingMethod, shippingAddress?: StripeAddress): Promise<void>;
+
+
+  /**
+   * Calls the client-implemented Stripe backend to confirm a Payment Intent (manual confirmation mode).
+   *
+   * @param paymentIntentID The Payment Intent ID to send to the backend.
+   * @returns a Promise that resolves on success and rejects on failure.
+   *     Any error should be reported as a string that can be displayed to the user.
+   */
+  confirmPaymentIntent(paymentIntentID: string): Promise<any>;
```

Comment:
Do we know what type it returns in the promise? I'd prefer not using `any` if it can be helped.

Reply:
Yes, that's a `StripeService` backend call - needs to be declared on the interface to be callable from the user flow.